src/containers/Identity/PersonalInfo/ClaimCategoryDetails/View.js

```diff
@@ -10,16 +10,23 @@ const ClaimCategoryDetails = (props) => {
   const {
     claims,
     navigation,
-    actions: { setActiveClaim },
+    showHiddenClaims,
+    actions: { setActiveClaim, setShowHiddenClaims },
   } = props;
 
   const goToClaimDetails = (claim) => {
     setActiveClaim(claim);
     navigation.navigate('ClaimDetails', { id: claim.get('id', ''), claimName: claim.get('name', '') });
   };
 
+
   return (
     <View style={styles.root}>
+      <CheckBox
+        checked={showHiddenClaims}
+        title="Show hidden claims"
+        onPress={() => setShowHiddenClaims(!showHiddenClaims)}
```

Comment:
If you are only using this for this kind of use case. You can name the action toggleHiddenClaims and do all the logic in the saga.